src/backend/renderer/gles2/mod.rs

```diff
@@ -1959,6 +1960,8 @@ impl Frame for Gles2Frame {
                 .DisableVertexAttribArray(self.solid_program.attrib_vert as u32);
             self.gl
                 .DisableVertexAttribArray(self.solid_program.attrib_position as u32);
+            self.gl.Enable(ffi::BLEND);
+            self.gl.BlendFunc(ffi::ONE, ffi::ONE_MINUS_SRC_ALPHA);
```

Comment:
I'd guess that disabling the blend test means you need to set the blend function again?

Reply:
Right. At least I am not sure, if it resets the function and I am rather explicit about it.